frontend/src/scenes/actions/ActionsTable.tsx

```diff
@@ -51,6 +52,12 @@ export function ActionsTable(): JSX.Element {
     const [filterByMe, setFilterByMe] = useState(false)
     const { user, hasAvailableFeature } = useValues(userLogic)
     const { featureFlags } = useValues(featureFlagLogic)
+    const { reportDataManagementActionDefinitionsPageViewed } = useActions(eventUsageLogic)
+
+    // Emit event once at component level until we create a logic for actions table that doesn't depend on actionsModel
+    useEffect(() => {
+        reportDataManagementActionDefinitionsPageViewed(-1, actions.length)
```

Comment:
A couple thoughts/questions here:
1. If you navigate directly to this page, this fires before actions loads, so it's always 0. Also, not sure how useful the action count is.
2. The load time being `-1` feels odd. Do we need to track load time on this event, and if so, seems like `-1` is going to throw off all of our metrics if we do average etc.
3. Do we need need this event or is it covered by the `pageview` event?

Reply:
Really good points here. I wanted to measure the load time for actions, but this metric feels useless since we don't encapsulate loading actions in a logic dedicated for this page. I'll remove this metric for now and maybe add it back later once it's refactored.